app/services/gcp_storage.py

```
@@ -0,0 +1,75 @@
+import os
+import uuid
+import logging
+from typing import Optional
```

Ruff (0.12.2)

`typing.Optional` imported but unused

Remove unused import: `typing.Optional`

(F401)